docs/source/kapsel/index.rst

```diff
@@ -182,7 +182,7 @@ The command line session will look something like::
 
 If you look at ``kapsel.yml`` now, you'll see bokeh and pandas
 listed under the ``packages:`` section. You will also see files
-such as ``envs/default/bin/bokeh`` since the packages have now
+such as ``envs/default/conda-meta/bokeh`` since the packages have now
```

Comment:
It looks like it's `Scripts\bokeh.bat` on Windows, if you want to do the platforms separately.